guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p><strong>Create a Folder</strong>: Create a folder and include the custom NLP function file. In our case, we will create a folder&nbsp;<code>NER_gilner_weather</code>. Inside this, we will place&nbsp;<code>NER_gilner_weather.emd</code>, the NLP function file&nbsp;<code>MyCustomNER.py</code>,&nbsp;and the directory containing the code of&nbsp;<code>GLiNER</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>updated</p>